paper-slider.html

```diff
@@ -644,11 +644,11 @@
         }
       },
 
-      _maxMarkersChanged: function(maxMarkers) {
-        if (!this.snaps) {
+      _maxMarkersChanged: function(maxMarkers, min, max, snaps) {
```

Comment:
Can you rename this method to `_updateMarkers()`? Seems more appropriate since the convention for simple single-property observers is `<property>Changed()` and this is no longer that.